linode/manager
# upcoming: [M3-7303] - Replace Managed summary charts with Recharts

## Description 📝
Replace Managed summary charts with Recharts
- CPU Usage chart
- Network Transfer chart
- Disk I/O chart

## Preview 📷
| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/linode/manager/assets/115299789/9b142e0c-d573-4b25-b519-53b462b6f0c8" /> | <video src="https://github.com/linode/manager/assets/115299789/db53e1d0-e878-4393-91b8-932a6dabcf25" /> |

## How to test 🧪
### Prerequisites
(How to setup test environment)
- Have an account that's Managed (you can do so via admin)

### Verification steps 
(How to verify changes)
- Go to `/managed/summary` and verify the updated UI and that there are no regressions in the graph, units, etc 
  - X and Y axis labels will not match exactly but the data from hovering over the graph should still be the same

## As an Author I have considered 🤔

*Check all that apply*
